src/ngx_http_lua_variable.c

```diff
@@ -395,8 +398,9 @@ ngx_http_lua_ffi_var_set(ngx_http_request_t *r, u_char *name_data,
     if (v) {
         if (!(v->flags & NGX_HTTP_VAR_CHANGEABLE)) {
             dd("variable not changeable");
-            ngx_snprintf(errbuf, errlen, "variable \"%*s\" not changeable",
-                         name_len, lowcase_buf);
+            *errlen = ngx_snprintf(errbuf, *errlen,
+                         "variable \"%*s\" not changeable",
```

Comment:
Style: the continued lines should be aligned up vertically with the first argument of the function call in the previous line. See the rest of the code base for such examples. Please also fix other similar places. Thank you!